client/navigation/components/header/index.js

```diff
@@ -15,6 +16,22 @@ const Header = () => {
 		document.body.classList.toggle( 'is-folded' );
 	};
 
+	const foldOnMobile = ( screenWidth ) => {
+		if ( screenWidth <= 960 ) {
+			document.body.classList.add( 'is-folded' );
+		}
+	};
+
+	useEffect( () => {
+		foldOnMobile( document.body.clientWidth );
+
+		window.addEventListener(
+			'orientationchange',
```

Comment:
Should this happen on resize as well?  The way the current WP menu changes also occurs on resize, but I can see a case being made for that not happening.

Reply:
I'm ok with leaving out the resize eventListener. It feels like something only devs do when doing code review 😄